docs/content/start/telemetry.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Quick Start
+menuPosition: 4
+codeCopyButton: true
+aliases:
+  - "/docs/get-started/telemetry/"
+---
+
+
+With thousands of developers using our framework to build their services and applications, telemetry is an essential part of maintaining and troubleshooting the health of the applications, so we are offering just that! We have integrated a way for the developers to set up a telemetry system by allowing them to create custom logging and handling logics to be passed in and called by our Fluid Framework telemetry pipeline.
```

Comment:
```suggestion
With thousands of developers using our framework to build their services and applications, telemetry is an essential part of maintaining and troubleshooting the health of the applications. Developers may set up a telemetry system which allows them to create custom logging and handling logic to be passed in and called by our Fluid Framework telemetry pipeline.
```
nit: tone here sounds more like a blog post about a new feature, whereas I think these docs are intended to be more perennial.  this would be my suggestion to change that tone

Reply:
good point! Will make the change